.circleci/config.yml

```diff
@@ -25,30 +14,43 @@ aliases:
     paths:
       - ~/.cache/yarn
 
+
+  - &restore_yarn_cache_node_14
+    keys:
+      - v1-yarn-cache-node_14-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache-node_14-{{ .Branch }}
+      - v1-yarn-cache-node_14
+
+  - &save_yarn_cache_node_14
+    key: v1-yarn-cache-node_14-{{ .Branch }}-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
   - &yarn_install
     name: Installing
     command: yarn --frozen-lockfile
 
 executors:
-  node_10:
-    docker:
-      - image: circleci/node:10.24.1
-    working_directory: *working_directory
   node_12:
     docker:
       # Keep it at `12.18.2` as otherwise there may be bugs with `graceful-fs` dependency
       # https://github.com/nodejs/node/issues/34529
+      # https://raw.githubusercontent.com/isaacs/node-graceful-fs/168bdb8f0bb3174e8499d4bc5878deead4172c39/polyfills.js
       - image: circleci/node:12.18.2
     working_directory: *working_directory
+  node_14:
+    docker:
+      - image: circleci/node:14.18.1
+    working_directory: *working_directory
 
 jobs:
-  install_test_node_10:
-    executor: node_10
+  install_test_node_12:
```

Comment:
I would also drop support for node v12 to be honest. The support goes EOL by end of April 2022.

https://nodejs.org/en/about/releases/